rcfiles/jointLklDM.rc

```diff
@@ -21,6 +21,13 @@ jointLklDM.plotmax: 1e-22
 # scale factor for results (e.g. to simulate for a Tobs and get results for another Tobs)
 # jointLklDM.plotScale: 1
 
+# Configuration for data exporter
+jointLklDM.exportData: FALSE
+jointLklDM.svMin: 1e-28
+jointLklDM.svMax: 1e-18
+jointLklDM.svNPoints: 1000
+jointLklDM.svLogStep: TRUE
+jointLklDM.logJ: 19.
```

Comment:
1st part of the comment: I rememeber the conversation and I agree. Before implementing anything I would prefer we submit the idea ti the rest of the GD group to get their feedback. Also keeping this version (hopefully just for now) would allow me to continue to work with the ICRC files. No problem on my side to keep this PR open until this question is solved.

2nd part of the comment: done.